programs/editor/wodotexteditor.js

```diff
@@ -298,6 +301,7 @@ var Wodo = Wodo || (function () {
             //
             eventNotifier = new core.EventNotifier([
                 EVENT_UNKNOWNERROR,
+                EVENT_MODIFIEDCHANGED,
```

Comment:
MODIFIEDCHANGED is a slightly confusing event name, as the words are synonyms (i.e., does this signal that the "modified" flag was changed, or was does it indicate that we modified the "changed" flag?).

Perhaps "PENDINGCHANGED" (i.e., pending changes)? Or, UNMODIFIEDCHANGED?